packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -176,44 +176,38 @@ const ClinicalResultBloodPressure = (props) => {
     const conceptDisplayElement = createConcpetDisplays(compareConceptDisplays);
 
     if (systolic || diastolic) {
-      if ((compareUnits.systolic === compareUnits.diastolic) && !compareStatusIsInError.systolic) {
+      if (!systolic) {
+        decoratedResultDisplay.push(<ResultError key={`Error-Systolic-${id}`} />);
+      }
+      if (noDataSystolic) {
```

Comment:
should this conditional be an `else if`? otherwise, the second `if` statement is always evaluated.
```suggestion
      else if (noDataSystolic) {
```

Reply:
it should be and i just pushed a fix for it. https://github.com/cerner/terra-clinical/pull/614/commits/44980eae8dbcfb4bbd0a0e918e308e9d12494a01#diff-13926e711e02256769a4612c0d911268R181